EternalBAND.Win/Controllers/WebApi/ErrorWebController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Azure;
+using EternalBAND.Api.Helpers;
+using EternalBAND.Api.Services;
+using EternalBAND.DomainObjects.ViewModel;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System.Net;
+
+namespace EternalBAND.Win.Controllers.WebApi
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ErrorWebController : ControllerBase
+    {
+        public ErrorWebController()
+        {
+        }
+
```

Comment:
public ErrorWebController()
        {
        }
        
is this constructor necessary? In Java If a class has no constructors, the Java compiler automatically provides a default constructor. This is called an implicit default constructor.

Reply:
Time to time at the IOC level if you register some class from interface it is giving compiler error but for controller it should be not a problem, I will prune and give a shot.